### DISCUSSION TITLE
迅雷已经填了账号密码，但是无法显示文件，你们有成功的吗

### DISCUSSION THREAD

**Discussion by raygzcn** [AlistGo/alist#807] [Q&A] [answered]
迅雷已经填了账号密码，但是无法登陆
按着教程来的，只填账号和密码，不成功。
pikpak也是按教程做，无问题

**Comment by 631894570** [comment]
迅雷网盘根目录未设置 登录正常 docker部署
![pic](https://pan.yccz.top:2443/d/%E6%9C%AC%E5%9C%B0%E7%A3%81%E7%9B%98/20220324110237.png)

**Comment by xhofe** [comment]
演示站正常啊。

**Comment by snow212-cn** [comment, marked as answer]
没有成功，一样的问题，之前开帖问过，别人都不搭理，你们有成功的吗
![image](https://user-images.githubusercontent.com/18443918/160057118-37a3ed19-ce17-4cfa-b274-b3f13e432832.png)
![image](https://user-images.githubusercontent.com/18443918/160057200-6e81766e-3fc6-4185-b6b1-d4c925dd4bdb.png)

**Reply by raygzcn** [reply]
也是一样的问题

**Comment by rakaki** [comment]
迅雷我倒是没问题
